common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -28,23 +28,16 @@ object FaciaContentFrontendHelpers {
     def mainYouTubeMediaAtom: Option[MediaAtom] =
       for {
        main <- faciaContent.properties.maybeContent.map(_.fields.main)
-       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       mediaAtoms <-  faciaContent.properties.maybeContent.map(_.elements.mediaAtoms)
        document <- Some(Jsoup.parse(main))
        atomContainer <- Option(document.getElementsByClass("element-atom").first())
        bodyElement <- Some(atomContainer.getElementsByTag("gu-atom"))
        atomId <- Some(bodyElement.attr("data-atom-id"))
-       mainMediaAtom <- atoms.media.find(ma => (ma.id == atomId && !ma.expired.getOrElse(false)) && ma.assets.exists(_.platform == MediaAssetPlatform.Youtube))
+       mainMediaAtom <- mediaAtoms.find(ma => (ma.id == atomId && !ma.expired.getOrElse(false)) && ma.assets.exists(_.platform == MediaAssetPlatform.Youtube))
      } yield mainMediaAtom
 
 
-    def mainVideo: Option[VideoElement] = {
-      val elements: Seq[Element] = faciaContent.properties.maybeContent.map(_.elements.elements).getOrElse(Nil)
-      val videos: Seq[VideoElement] = elements.flatMap {
-        case video: VideoElement => Some(video)
-        case _ => None
-      }
-      videos.find(_.properties.isMain)
```

Comment:
We store the main video in the `PressedStory` elements: See https://github.com/guardian/frontend/blob/952bed88badab250f3bee0a470adb32212175d23/common/app/model/pressedContent.scala#L165-L165

This logic was already there in the `Elements` object https://github.com/guardian/frontend/blob/8c8c3f5b6bdebf8fe3bb80f73de1e0d773099259/common/app/model/meta.scala#L574 and https://github.com/guardian/frontend/blob/8c8c3f5b6bdebf8fe3bb80f73de1e0d773099259/common/app/model/meta.scala#L604